crates/modelardb_bulkloader/src/main.rs

```diff
@@ -52,14 +52,22 @@ async fn main() -> Result<()> {
         print_usage_and_exit_with_error();
     }
 
-    // unwrap() is safe as the path of the binary is dropped.
```

Comment:
@CGodiksen and @chrthomsen open question, should we keep all the `unwrap()` comments and just replace `unwrap()` with `expect()`. Since the `expect()` messages seem to inform the user about what went wrong, while the comments seem to inform the developer what invariance should prevent the `expect()` from ever being triggered both seem useful.